src/AvPortalResource.php

```diff
@@ -100,20 +104,52 @@ public function getThumbnailUrl(): ?string {
       return NULL;
     }
 
+    if ($this->getType() == 'VIDEO') {
+      return $this->getVideoThumbnailUrl();
+    }
+    elseif (in_array($this->getType(), ['PHOTO', 'REPORTAGE'])) {
+      return $this->getPhotoThumbnailUrl();
+    }
+
+    return NULL;
+  }
+
+  /**
+   * Returns the thumbnail of a video resource.
+   *
+   * @return string|null
+   *   URL of the thumbnail.
+   */
+  protected function getVideoThumbnailUrl(): ?string {
+    $first_media_json = reset($this->data['media_json']);
+
+    // If available, get a language agnostic thumbnail.
+    if (isset($first_media_json['INT']['THUMB'])) {
```

Comment:
Let's add an extra fallback here. After checking if the international version is available, check for the current site language. Then return the first available in the languages array.